modules/apps/depot/depot-web/src/main/java/com/liferay/depot/web/internal/info/permission/provider/DepotEntryInfoPermissionProvider.java

```diff
@@ -0,0 +1,42 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.depot.web.internal.info.permission.provider;
+
+import com.liferay.depot.model.DepotEntry;
+import com.liferay.info.permission.provider.InfoPermissionProvider;
+import com.liferay.portal.kernel.feature.flag.FeatureFlagManagerUtil;
+import com.liferay.portal.kernel.security.permission.PermissionChecker;
+
+import org.osgi.service.component.annotations.Component;
+
+/**
+ * @author Adolfo Pérez
+ */
+@Component(service = InfoPermissionProvider.class)
+public class DepotEntryInfoPermissionProvider
+	implements InfoPermissionProvider<DepotEntry> {
+
+	@Override
+	public boolean hasAddPermission(
+		long groupId, PermissionChecker permissionChecker) {
+
+		if (FeatureFlagManagerUtil.isEnabled("LPD-17564")) {
```

Comment:
is it possible to add the company id to check if the FF is enabled? maybe can use permissionChecker.getCompanyId()

Reply:
Done.